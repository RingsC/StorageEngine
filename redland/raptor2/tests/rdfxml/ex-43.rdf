<?xml version="1.0"?>
<rdf:RDF xmlns:ex="http://example.org/ns#"
         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xml:base="http://example.org/triples">

  <!-- rdf:bagID where a generated ID is needed  -->

  <rdf:Description rdf:about="http://example.org/node" rdf:bagID="bag">
    <ex:prop1>val1</ex:prop1>
  </rdf:Description>

</rdf:RDF>
