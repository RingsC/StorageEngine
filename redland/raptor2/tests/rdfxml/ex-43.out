<http://example.org/triples#bag> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Bag> .
<http://example.org/node> <http://example.org/ns#prop1> "val1" .
<http://example.org/triples#bag> <http://www.w3.org/1999/02/22-rdf-syntax-ns#_1> _:genid1 .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Statement> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#subject> <http://example.org/node> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#predicate> <http://example.org/ns#prop1> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#object> "val1" .
